pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -117,6 +137,7 @@ func (l *listUsersQuery) ListUsers(
 	for foundUser := range foundUsersUnique {
 		foundUsers = append(foundUsers, tuple.StringToUserProto(foundUser))
 	}
+	span.SetAttributes(attribute.Int("result_count", len(foundUsers)))
```

Comment:
**Note for future:** We'll also need another attribute for "exceptions" in #1534